docs/tasks/debug-application-cluster/audit.md

```diff
@@ -72,6 +72,9 @@ In both cases, audit events structure is defined by the API in the
 `audit.k8s.io` API group. The current version of the API is
 [`v1beta1`][auditing-api].
 
+**Note:** in case of patches, request objects are arrays, containing operations, not the appropriate
+API objects. You should design your system to behave correctly in this case.
```

Comment:
> in case of patches, request objects are arrays, containing operations, not the appropriate API objects. You should design your system to behave correctly in this case.

Sorry, in fact, I don't understand this sentence at all....:crying_cat_face: 
What's the relation between audit objects and  `patches`?
When would we patch an audit object?

Reply:
Maybe an example would help?

Reply:
Added an example